data/korath/korath missions.txt

```diff
@@ -576,3 +576,621 @@ mission "Efreti: Distress Call 8"
+			action
+				log "Failed to land on Rabasot Pa'ah due to a powerful atmospheric event brought on by unusual interactions between the planet's atmosphere and the nova it orbits. Reconfigured the shields to act as a plow to get out of the storm. That shield configuration should allow safe transit through the atmosphere in another landing attempt."
+			`As you descend through the magnetosphere, your computer warns you of a build-up of unusual charged particles called muons, which is a bit odd. From your training on hyperspace exit and entrance, you recall that the mean lifetime of a muon is measured in microseconds. Something here must be actively generating them.`
+			`	You travel deeper into the atmosphere, approaching what appears to be an inactive volcanic island chain. As usual, your ship's descent causes air currents, but the strange atmosphere turns those currents into a powerful storm. The muons mix with this event, creating catastrophic explosions of electrical discharge.`
+			`	Remembering what you've read about extreme atmospheric landings, you reconfigure your shields to act as a plow, pushing through the storm. At first it isn't enough, so you pull upward, hoping the lower atmospheric density will let you escape. Once you're free, you realize you've already reached the exosphere.`
+			`	A quick analysis by your ship's computer tells you the shield plow technique should get you all the way down to the surface, if you avoid eddies in the magnetosphere. It seems those eddies interact with the carbon star in an odd manner that produces vast amounts of muons. Your shield was drawing them in and discharging them, creating an intense storm centered on your ship. The plow shape directed the electrical activity aft, letting you move freely.`
+			`	Certainly, a second landing attempt will get you through the atmosphere. Next, you'll have to scout for a safe place to land on this world of water and storms.`
+				launch
+
+			label "second attempt"
+			scene scene/volcanic-eruption-at-night
+			action
+				log "Was able to pass through the atmosphere of Rabasot Pa'ah despite the unusual atmospheric event. A volcanic eruption nearly hit the ship, perhaps caused by landing in the wrong area. There is a large island that might be suitable for landing. Another attempt could succeed."
+			`In this trip, your ship's descent stirred up awe-inspiring storm clouds, but traveling through them was easy. Your shield plow technique took you down to the lower troposphere, where the storms cleared enough for you to see. The original island chain was too far away, so you set course for another one nearby.`
+			`	Aiming for the hardest-looking piece of ground your sensors can find, you touch down. As this planet has twice Earth's gravity, you extend your ship's artificial gravity field to the surface. Donning a space suit for safety's sake, you walk out to see nature taking control once again: the massive volcano in front of you starts to erupt.`
```

Comment:
```suggestion
			`	Aiming for the hardest-looking piece of ground your sensors can find, you touch down. Noting the extensive radiation, you extend your ship's shields to the surface. Donning a space suit for safety's sake, you walk out to see nature taking control once again: the massive volcano in front of you starts to erupt.`
```

This is the anti-gravity reference I was referring to.

Reply:
I see what you mean.

I was referring to the simple gravity deck plating, not anything fancy. I'll get rid of that.